CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 107.0.0
+
+In this major release we deleted the deprecated themer classes: `MDCAlertScheme` and `MDCAlertControllerThemer`.
+
+## Breaking changes
+
+Deleted `MDCAlertScheme` and `MDCAlertControllerThemer` class, please use the theming extensions.
+Learn more at https://github.com/material-components/material-components-ios/blob/develop/docs/theming.md#migration-guide-themers-to-theming-extensions
+
+## New features
+
+In this release we added the ability to be informed about presentation events of a `MDCAlertController` as well as various minor improvements to the `Dialogs` component and other bug fixes.
+
+## API changes
+
+### Dialogs
+
+*new* property: `delegate` in `MDCAlertController`
+
+*new* protocol: `MDCAlertControllerDelegate`
```

Comment:
You can add something along these lines:
"The new delegate MDCAlertControllerDelegate protocol exposes dialogs life cycle events that are useful for impression logging."

Reply:
We typically don't describe the APIs functionality in the API section, I've updated the _new features_ to be along these lines.